content/docs/04.5-devx/05-app-profile.md

```diff
@@ -14,13 +14,13 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 
 
-## App Profiles:
+## App Profiles
 
 App Profiles are templates created with preconfigured services required for sandbox cluster deployment. App Profiles provide a way to drive consistency across sandbox clusters. You can create as many profiles as required. An App Profile can be created to meet specific types of Workload on Palette Sandbox Clusters. For example, you may create a Development App Profile with Helm Charts, Manifest, different database applications, etc. 
 
 <br />
```

Comment:
Lets mention about two sets of parameters:

1. Input parameters - What user can specify and will be consumed by the current tier. It can even use output variables from below tier.
2. Output parameters - which will be generated once the tier gets deployed. It can be consumed by the above tiers.